lib/kafka_gen_stage/consumer.ex

```diff
@@ -154,15 +141,14 @@ defmodule KafkaGenStage.Consumer do
           consumer: pid(),
           consumer_ref: reference(),
           queue: :queue.queue(),
+          high_wm_offset: non_neg_integer() | atom(),
```

Comment:
Why not `:unknown` or perhaps better still `nil`? Can we get an atom from brod?

Reply:
OK, will use `nil`